nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -535,10 +535,17 @@ in
       '';
     };
 
+    isoImage.makeBiosBootable = mkOption {
+      default = false;
+      description = lib.mdDoc ''
+        Whether the ISO image should be a BIOS-bootable disk.
+      '';
+    };
+
     isoImage.makeEfiBootable = mkOption {
       default = false;
       description = lib.mdDoc ''
-        Whether the ISO image should be an efi-bootable volume.
+        Whether the ISO image should be an EFI-bootable volume.
```

Comment:
This is an unrelated change, please put it in another commit or drop it from this PR (otherwise it would mess the blame)

Reply:
Done, this change is now in separate commit.